01-Navigation/4-Contextual_navigation/4.6-Navigate_To_menu_tools.cs

```diff
@@ -5,9 +5,9 @@
 //
 // Very useful way of navigating without having to learn ALL of the shortcuts!
 //
-// Alt+` (VS)
-// Ctrl+Shift+G (IntelliJ)
-
+// <shortcut id="Navigate To...">Alt+`</shortcut>
+//
+// TODO: Not sure this is relevant anymore
```

Comment:
Sure is, at least locate in SLN explorer is still there, as well as file nearby and all others. Commandline and explorer seem gone, so they can be dropped.

![image](https://user-images.githubusercontent.com/485230/80714207-dea17d00-8af4-11ea-9a5c-729dfb209216.png)